### PR TITLE
Fixed an error inside component.php file, coming from an old change.

### PR DESCRIPTION
**Pull request by gpongelli**
Added sqlValue method inside PostgreSQL's driver, added its use inside insertObject and inside updateObject.
Added sqlValue's test.

### CODE REVIEW

File: tests/suites/database/driver/postgresql/JDatabasePostgresqlTest.php [hunk 1]
```diff
@@ -580,7 +580,7 @@ public function testInsertObject()
 		$tstK->title = "PostgreSQL test insertObject with key";
 		$tstK->start_date = '2012-04-07 15:00:00';
 		$tstK->description = "Test insertObject with key";
-		$retK = self::$driver->insertObject('#__dbtest', $tstK, 'id');
+		$retK = self::$driver->insertObject('jos_dbtest', $tstK, 'id');
```
Comment (realityking) [inline tests/suites/database/driver/postgresql/JDatabasePostgresqlTest.php]: Why did you make this change?

Comment (gpongelli) [inline tests/suites/database/driver/postgresql/JDatabasePostgresqlTest.php]: I changed it for an error that was fixed, but I've forgotten it.
I'm changing this.

Comment (gpongelli) [inline tests/suites/database/driver/postgresql/JDatabasePostgresqlTest.php]: Fixed.

File: libraries/joomla/installer/adapters/component.php [hunk 2]
```diff
@@ -480,7 +480,7 @@ public function install()
 			return false;
 		}
 
-		$eid = $row->$key;
+		$eid = $row->extension_id;
```
Comment (LouisLandry) [inline libraries/joomla/installer/adapters/component.php:483]: This seems somewhat random for this pull request.  Can you explain what this is for?

Comment (LouisLandry) [inline libraries/joomla/installer/adapters/component.php:483]: Nevermind, I looked a little deeper.

**Pull request merged**